src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -84,7 +91,11 @@ public void initialized() {
         initialized = true;
 
         if (gerritPushTargetPanels.size() == 1) {
-            branchTextField.setText(gerritPushTargetPanels.values().iterator().next());
+            String branchName = gerritPushTargetPanels.values().iterator().next();
+
+            String gitReviewBranchName = getGitReviewBranchName();
```

Comment:
Try to return a Guava Optional here, instead of null.

Reply:
Implemented optional approach for returning gitreview branch name. I was lack of experience with Guava before, so didn't know that there is a possibility to use optional. If it was java 8 I would certainly prefer using built-in optional.